internal/website/content/howto/server/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Server"
+date: 2019-06-21T10:36:43-07:00
+draft: true
+showInSidenav: true
+---
+
+The Go CDK's `server` package provides a pre-configured HTTP server with diagnostic hooks for request logging, health checks, and trace exporting via OpenCensus. These guides will show you how to start up and shut down the server, as well as how to work with the request logging, health checks, and trace exporting.
+
+The Go CDK includes a server package because??
```

Comment:
I suggest you turn this into a real paragraph for now (I'm concerned we will forget it in its current state - there's a nonzero chance...) and reviewers will suggest fixes if interested